firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/SignInStorage.java

```diff
@@ -54,12 +57,14 @@ Task<Boolean> getSignInStatus() {
   }
 
   boolean getSignInStatusBlocking() {
```

Comment:
Should this method be renamed?

Reply:
No I don't think so. It still returns the sign in status, and it is still potentially blocking.

I changed `getSharedPreferencesBlocking()` to `getAndCacheSharedPreferences()` because if I included `Blocking` it would be super long and I didn't think it was necessary for the private method name to communicate that it blocks.